unit_tests/sources/declarative/test_concurrent_declarative_source.py

```diff
@@ -479,20 +479,19 @@ def test_group_streams():
     synchronous_streams = source._synchronous_streams
 
     # 2 incremental streams
```

Comment:
nit: should update these comments to reflect 2 incremental, 1 full refresh as concurrent and 1 substream as synchronous